modules/extensions-spring-support/src/test/java/org/mule/module/extension/internal/ImplicitConfigTestCase.java

```diff
@@ -64,7 +64,7 @@ public void getImplicitConfig() throws Exception
 
     @Extension(name = "implicit")
     @Operations({ImplicitOperations.class})
-    @Xml(schemaLocation = "http://www.mulesoft.org/schema/mule/implicit", namespace = "implicit", schemaVersion = "1.0")
+    @Xml(namespaceLocation = "http://www.mulesoft.org/schema/mule/implicit", namespace = "implicit", schemaVersion = "1.0")
```

Comment:
If we use the schema imports as a rule, then we have:

```<xs:import 
namespace="http://www.mulesoft.org/schema/mule/implicit" schemaLocation="http://www.mulesoft.org/schema/mule/extension/current/mule-implicit.xsd"></xs:import>```

So changing that URI to `namespaceLocation` is less confusing